### PR TITLE
WRP-11631: Changed dynamic color change to run on the entire app

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
 Changed dynamic color change to run on the entire app

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 Changed dynamic color change to run on the entire app

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-11631

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

File: console/src/App/App.js [hunk 1]
```diff
@@ -289,7 +410,7 @@ const AppBase = kind({
 						onReloadApp={reloadApp}
 						onToggleDateTimePopup={onToggleDateTimePopup}
 					/>
-					<ThemeSettings onSelect={onSelect} onSendSkinSettings={sendSkinSettings} prevIndex={prevIndex} />
+					<ThemeSettings onSelect={onSelect} onSendSkinSettings={sendSkinSettings} onToggleDynamicColor={onToggleDynamicColor} onToggleRealTime={handleRealState} realTime={realTime} prevIndex={prevIndex} />
```
Comment (stanca-pop-lgp) [inline console/src/App/App.js]: Please sort props.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]

File: console/src/App/App.js (hunk 1 above)
Comment (daniel-stoian-lgp) [inline console/src/App/App.js]:
```suggestion
					<ThemeSettings onSelect={onSelect} onSendSkinSettings={sendSkinSettings} onToggleDynamicColor={onToggleDynamicColor} onToggleRealTime={handleRealState} prevIndex={prevIndex} realTime={realTime} />
```

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**